querybook/webapp/components/QueryExecution/SamplingToolTip.tsx

```diff
@@ -12,17 +12,19 @@ interface SamplingTooltipProps {
     queryExecution: IQueryExecution;
     onSamplingInfoClick?: () => void;
     hasSamplingTables?: boolean;
+    sampleRate: number;
 }
 
 export const SamplingTooltip: React.FC<SamplingTooltipProps> = ({
     queryExecution: { status, id },
     onSamplingInfoClick,
     hasSamplingTables,
+    sampleRate,
 }) => {
     const { enabled, sampling_tool_tip_delay: delay } =
         PublicConfig.table_sampling;
 
-    const queryCanBeSampled = enabled && hasSamplingTables;
+    const queryCanBeSampled = enabled && hasSamplingTables && sampleRate <= 0;
```

Comment:
nit: the name `queryCanBeSampled` here seems inappropriate anymore

Reply:
@jczhong84 how about `isQuerySamplingEligible`?

Reply:
or just remove the variable here but check them directly inside the useEffect
```
if(enabled && hasSamplingTables && sampleRate <= 0 && status <= QueryExecutionStatus.RUNNING`)
```